lib/config.js

```diff
@@ -229,6 +231,16 @@ function getLocalConfig(thisConfig, directory) {
             continue;
         }
 
+        // Check for root flag
+        if (localConfig.root === true) {
+            rootPath = path.dirname(localConfigFile);
+        }
+
+        // If root flag is set, don't consider file if it is above root
+        if (rootPath && !pathIsInside(path.dirname(localConfigFile), rootPath)) {
```

Comment:
I'm just trying to follow the API of the `path-is-inside` module.

Usage from `path-is-inside` README:

> First the path being tested; then the potential parent.

I don't know if full filename will work as expected, so I figured it doesn't hurt to normalize to just consider paths (since that's all we care about in this case anyway).